src/main/java/org/beanpod/switchboard/entity/OutputChannelEntity.java

```diff
@@ -31,17 +30,14 @@
   @Column(name = "id")
   private Long id;
 
-  @OneToOne(
-      optional = false,
```

Comment:
Why are we removing this? We shouldn't have an input or output channel that doesn't have an associated channel object right?

Reply:
I agree with you, however if i have the optional tag, I would get an error like: not-null property references a null or transient value.
This is part of the workaround solution where the logic doesn't really meet the code.